examples/meter/meter.js

```diff
@@ -0,0 +1,32 @@
+/** @type {!Element} Meter element. */
+let meterEl;
+/** @type {!Element} Element visually filling the meter. */
+let visualFillEl;
+/** @type {number} Minimum value. */
+let minUsage;
+/** @type {number} Maximum value. */
+let maxUsage;
+/** @type {number} Current value. */
+let usage;
+
+/** Set the value randomly between the minimum and maximum. */
+function randomizeUsage() {
+  const range = maxUsage - minUsage;
+  usage = Math.floor((Math.random() * range) + minUsage);
+  meterEl.setAttribute('aria-valuenow', usage);
+  render();
+}
+function render() {
+  const asPercentage = (100 * usage) / (maxUsage - minUsage);
+  visualFillEl.style.width = asPercentage + '%';
+}
+function init() {
+  meterEl = document.querySelector('[role=meter]');
```

Comment:
Would it be worth it to use `document.querySelectorAll('[role=meter]')`, then loop through and initiate them, to allow for the possibility of multiple meters per page?

Reply:
That's a great practice for writing libraries where usage isn't controlled by the author, but we also control the content on this page so I don't think it's exactly necessary.

If we ever decide to add another meter to the page, the `init` method can be updated as part of that PR.

Reply:
Thanks for your feedback @smhigley and @sh0ji

Reply:
I'd agree with this, if the only issue were `document.querySelector` vs `querySelectorAll` + a loop to call an init function:
> That's a great practice for writing libraries where usage isn't controlled by the author, but we also control the content on this page so I don't think it's exactly necessary.

My main concern is actually that the state for the meter component is being stored in one-off global variables, so if more than one meter were needed, all the logic would need to be rewritten. If we move away from this to a constructor function or class-like approach, then I wouldn't have an issue with not looping and `init`-ing multiple possible meters.

Reply:
Yeah, you're right. I meant to convey that this isn't breaking anything now, so I don't think it should block merging. @a11ydoer is of course more than welcome to change it now.

Keep in mind that we'll need to do a large-scale refactor later if we switch to ES6, so that would also be an appropriate time to add the loop, as well as just skipping the constructor function and going straight to a class.